cloud_deploy/nodes.py

```diff
@@ -107,10 +107,10 @@ def create(cls, name, type="docker", size="512mb"):
                 region='ams2',
                 image=type,
                 size_slug=size,
-                ssh_keys=['66:0b:b5:20:a0:68:f9:fc:82:5a:de:c1:ce:03:4f:84'])
+                ssh_keys=['66:0b:b5:20:a0:68:f9:fc:82:5a:de:c1:ce:03:4f:84']) #think about what to do with that
```

Comment:
Perhaps we should create a config file, where a list of ssh public keys could be included. In the meantime, perhaps just add your key fingerprint to this list
(run `ssh-keygen -E md5 -lf /path/to/your_key.pub` to generate the fingerprint)